src/main/java/core/basesyntax/service/FileRead.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface FileRead {
```

Comment:
Please stick to the naming convention of the interfaces. Follow the convention across the all files, methods, variables. Fix the same issue in all places
```suggestion
public interface FileReader {
```